plugins/BEdita/Core/src/Model/Action/ListEntitiesAction.php

```diff
@@ -138,6 +140,13 @@ protected function buildFilter(Query $query, array $filter)
 
                 continue;
             }
+
+            // No suitable filter was found
+            //$this->log('Filter not found ' . $key, 'error');
+            throw new BadRequestException([
```

Comment:
Agreed, I had same doubt
Probably a custom exception is the best solution
`LogicException` from PHP guide: <<_Exception that represents error in the program logic. This kind of exception should lead directly to a fix in your code._>>